src/data_provider/src/packages/rpmlib.h

```diff
@@ -0,0 +1,89 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
```

Comment:
MIssing preprocessor guards
```suggestion
#ifndef _RPMLIB_H
#define _RPMLIB_H

```